soh/soh/SaveManager.cpp

```diff
@@ -124,7 +130,13 @@ void SaveManager::LoadRandomizerVersion2() {
 
     SaveManager::Instance->LoadArray("itemLocations", RC_MAX, [&](size_t i) {
         gSaveContext.itemLocations[i].check = RandomizerCheck(i);
-        SaveManager::Instance->LoadData("", gSaveContext.itemLocations[i].get);
+        SaveManager::Instance->LoadStruct("", [&]() {
+            SaveManager::Instance->LoadData("rgID", gSaveContext.itemLocations[i].get.rgID);
+            SaveManager::Instance->LoadData("fakeRgID", gSaveContext.itemLocations[i].get.fakeRgID);
+            std::string trickName;
+            SaveManager::Instance->LoadData("trickName", trickName);
+            strncpy(gSaveContext.itemLocations[i].get.trickName, trickName.c_str(), 50);
```

Comment:
just realized i should have said something about this earlier, 50 is a magic number here, it'd be good to `#define` it somewhere